spec/linter-phpcs-spec.js

```diff
@@ -16,15 +16,13 @@ const emptyPath = path.join(__dirname, 'files', 'empty.php');
 const longCP1251Path = path.join(__dirname, 'files', 'long.cp1251.php');
 const shortCP1251Path = path.join(__dirname, 'files', 'short.cp1251.php');
 
-function throwingLint(editor) {
```

Comment:
I'm not sure why this was even written like that, it's _much_ cleaner with pure `async`/`await` syntax.